docs/en/index.md

```diff
@@ -3,73 +3,64 @@ title: Home
     </div>
     <div class="col-6">
-        <h3 class="u-hide--small" style="visibility: hidden;">&nbsp;</h3>
-        <p>Finally, reference it from your own Sass files, with optional settings:</p>
-        <pre><code>// Optionally override some settings
-$color-brand: #ffffff;
-
-// Import the framework
-@import 'vanilla-framework/scss/vanilla';
+        <p>You can now reference Vanilla from your main Sass file - note that the path to <code>node_modules</code> might be different for your project:</p>
+        <pre><code>// Import the framework
+@import 'node_modules/vanilla-framework/scss/vanilla';
 
 // Include all of Vanilla Framework
 @include vanilla;</code></pre>
-        <p><em>Note: If you don't need the whole framework, you can just include specific parts, e.g. <code>@include vf-b-forms</code>.</em></p>
+        <p><em>For information on overriding settings, importing only parts of Vanilla, and build tooling, see <a href="/en/advanced-usage">Advanced usage</a>.</em></p>
     </div>
 </div>
 <hr class="is-deep">
 <div class="u-equal-height">
-    <div class="col-6">
+    <div class="col-12">
         <h3>Hotlink</h3>
-        <p>Alternatively, you an add Vanilla directly to your markup:</p>
-        <p><code>&lt;link rel="stylesheet" href="https://assets.ubuntu.com/v1/vanilla-framework-version-1.7.1.min.css" /&gt;</code></p>
-    </div>
-    <hr class="is-deep u-hide--medium u-hide--large">
-    <div class="col-6">
-        <h3>Download</h3>
-        <p>Download the latest version of Vanilla from <a href="https://github.com/vanilla-framework/vanilla-framework/releases/" class=" p-link--external">GitHub</a></p>
+        <p>You can add Vanilla directly to your markup:</p>
+        <pre><code>&lt;link rel="stylesheet" href="https://assets.ubuntu.com/v1/vanilla-framework-version-1.7.1.min.css" /&gt;</code></pre>
     </div>
 </div>
 <hr class="is-deep">
 <div class="u-equal-height">
-    <div class="col-6">
-        <h3>Using with GitHub Pages via Git submodules</h3>
-        <p>Creating a submodule in the git repo does not add all the code to the project but includes a reference and path to include the framework. You may find this method useful if you're planing to host on GitHub Pages.</p>
-    </div>
-    <div class="col-6">
-        <p>Run this command at the root of your project (replacing vX.X.X with the release you wish to use)</p>
-        <pre><code>git submodule add -b vX.X.X -- git@github.com:vanilla-framework/vanilla-framework.git _sass/vanilla-framework</code></pre>
-        <p>By running the following command it will pull down the framework into the correct location:</p>
-        <pre><code>git submodule update</code></pre>
+    <div class="col-12">
+        <h3>Download</h3>
+        <p>Download the latest version of Vanilla from GitHub.</p>
+        <a href="https://github.com/vanilla-framework/vanilla-framework/archive/v1.7.1.zip" class="p-button--positive">Download v1.7.1</a>
```

Comment:
Isn't this likely to go out of date?

Reply:
@anthonydillon Yeah 100% without a doubt. I'll create an issue for automating updating the docs with the latest version